.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I would say .cache since it added it because of my Linux machine, not node_modules. No node_modules as far as I'm aware will be in the root